master/buildbot/test/fake/docker.py

```diff
@@ -41,8 +45,9 @@ def build(self, fileobj, tag):
             self._images.append({'RepoTags': [tag + ':latest']})
 
     def create_host_config(self, *args, **kwargs):
-        pass
+        self.call_args_create_host_config.append(kwargs)
 
-    def create_container(self, image, *args, **kwargs):
+    def create_container(self, *args, **kwargs):
```

Comment:
I guess it was explicitly mentioned because image is the only positional argument in the real docker.Client API and all other arguments to create_container are keyworded arguments. So, it would be feasible now to call create_container like this `create_container(1, image=some_image)`, whereas before this was not possible. I will revert the change once we clarified the other issues.